ifrs17-template/Test/IfrsVariablesTest.ipynb

```diff
@@ -236,53 +325,79 @@
         {
             "cell_type": "code",
             "source": [
-                "var errorList = new List<BenchmarkTestResult>();",
-                "\nforeach (var g in bmFiles.GroupBy(x => (ReportingNode: x.ReportingNode, Year: x.Year, Month: x.Month, Scenario : x.Scenario)))",
+                "public async Task<List<BenchmarkTestResult>> CheckIfrsVariablesValuesAsync(BenchmarkMetadata[] bmFiles)",
                 "\n{",
-                "\n    var reportingNode = g.Key.ReportingNode;",
-                "\n    var year = g.Key.Year;",
-                "\n    var month = g.Key.Month;",
-                "\n    var scenario = g.Key.Scenario;",
+                "\n    var errorList = new List<BenchmarkTestResult>();",
+                "\n    foreach (var g in bmFiles.GroupBy(x => (ReportingNode: x.ReportingNode, Year: x.Year, Month: x.Month, Scenario : x.Scenario)))",
+                "\n    {",
+                "\n        var reportingNode = g.Key.ReportingNode;",
+                "\n        var year = g.Key.Year;",
+                "\n        var month = g.Key.Month;",
+                "\n        var scenario = g.Key.Scenario;",
                 "\n    ",
-                "\n    await DataSource.Partition.SetAsync<PartitionByReportingNodeAndPeriod>(new {ReportingNode = reportingNode, Year = year, Month = month, Scenario = scenario});",
-                "\n    var partition = (await DataSource.Query<PartitionByReportingNodeAndPeriod>().ToArrayAsync())",
-                "\n                        .SingleOrDefault(x => x.ReportingNode == reportingNode && x.Year== year && x.Month == month && x.Scenario == scenario)?.Id ?? ",
-                "\n                        throw new Exception(\"Partition not found.\");",
+                "\n        await DataSource.Partition.SetAsync<PartitionByReportingNodeAndPeriod>(new {ReportingNode = reportingNode, Year = year, Month = month, Scenario = scenario});",
+                "\n        var partition = (await DataSource.Query<PartitionByReportingNodeAndPeriod>().ToArrayAsync())",
+                "\n                            .SingleOrDefault(x => x.ReportingNode == reportingNode && x.Year == year && x.Month == month && x.Scenario == scenario)?.Id ?? ",
+                "\n                            throw new Exception(\"Partition not found.\");",
                 "\n    ",
-                "\n    var computedVariablesByEstimateType = (await DataSource.Query<IfrsVariable>()",
+                "\n        var computedVariablesByEstimateType = (await DataSource.Query<IfrsVariable>()",
                 "\n                                                            .Where(v => !excludedDataNodes.Contains(v.DataNode))",
+                "\n                                                            .Where(v => ProjectionCount > 1",
```

Comment:
Instead of a specific logic around projection, pls implement a positive filter and pass the filters: excludedDataNodes and includedDataNodes as input to this method

Reply:
Done.